scripts/roles.json

```diff
@@ -3130,11 +3379,11 @@
       "aria-keyshortcuts",
       "aria-label",
       "aria-labelledby",
+      "aria-level",
```

Comment:
Did the level 2 default change?

Reply:
https://www.w3.org/TR/wai-aria-1.2/#heading

Reply:
See https://github.com/A11yance/aria-query/pull/65#discussion_r449864490